### PR TITLE
Add improved UX for implemented commands

### LINKED ISSUE #132: CLI UX
Our CLI needs a great UX!

Commands that need UX
1. Login/Logout
2. Variable
3. List
4. Policy
5. User/Host

DOD
Login/logout
- [x] Command UX has been implemented as per Sharon's pages
- [x] Command UX for failure and interactive options are according to epic definition
- [x] Command UX has been demoed and approved by UX + PM/PO

Variable
- [x] Command UX has been implemented as per Sharon's pages
- [x] Command UX for failure and interactive options are according to epic definition
- [x] Command UX has been demoed and approved by UX + PM/PO

List
- [x] Command UX has been implemented as per Sharon's pages
- [x] Command UX for failure and interactive options are according to epic definition
- [x] Command UX has been demoed and approved by UX + PM/PO

Policy
- [x] Command UX has been implemented as per Sharon's pages
- [x] Command UX for failure and interactive options are according to epic definition
- [x] Command UX has been demoed and approved by UX + PM/PO

User/host
- [x] Command UX has been implemented as per Sharon's pages
- [x] Command UX for failure and interactive options are according to epic definition
- [x] Command UX has been demoed and approved by UX + PM/PO

Help
- [x] Command UX has been implemented as per Sharon's pages
- [x] Command UX for failure and interactive options are according to epic definition
- [x] Command UX has been demoed and approved by UX + PM/PO

Helpful resources:
UX design: https://ljfz3b.axshare.com/#id=c63xjv&p=commands_matrix___rules&g=1
helphttps://ljfz3b.axshare.com/#id=x8ktq8&p=conjur_help__init&g=1

### PR DESCRIPTION
**Pull request by sigalsax**
### What does this PR do?
This PR implements the UX according to the guidelines described here https://ljfz3b.axshare.com/#id=x8ktq8&p=conjur_help__init&g=1

To review, pack the CLI `pyinstaller -D ./pkg_bin/conjur` and launch commands according to the UX in above link

### What ticket does this PR close?
Resolves #132 

#### Change log
- [x] The CHANGELOG has been updated, or
- [ ] This PR does not include user-facing changes and doesn't require a CHANGELOG update

#### Test coverage
- [ ] This PR includes new unit and integration tests to go with the code changes, or
- [x] The changes in this PR do not require tests

#### Documentation
- [ ] Docs (e.g. `README`s) were updated in this PR, and/or there is a follow-on issue to update docs, or
- [x] This PR does not require updating any documentation

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,6 +16,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   [cyberark/conjur-api-python3#89](https://github.com/cyberark/conjur-api-python3/issues/89)
 
 ### Changed
+- CLI command UX has been improved according to UX guidelines
```
Comment (sigalsax) [inline CHANGELOG.md:19]: @orenbm how do we feel about this changelog message?

File: CHANGELOG.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   [cyberark/conjur-api-python3#101](https://github.com/cyberark/conjur-api-python3/issues/101)
 - The `user` methods rotate-api-key and change-password are now available in CLI and API to manage users
   [cyberark/conjur-api-python3#101](https://github.com/cyberark/conjur-api-python3/issues/101)
-- Add ability to get previous variable/secret versions. This is available in both CLI and API.
+- Add ability to get previous variable/secret versions. This is available in both CLI and API
```
Comment (orenbm) [inline CHANGELOG.md:14]: please add a link to an issue.

also - what does "Available in the API" mean? The Conjur REST API?

File: CHANGELOG.md (hunk 1 above)
Comment (orenbm) [reply CHANGELOG.md:23]: I think it's good. the link below has the needed explanations

File: CHANGELOG.md [hunk 3]
```diff
@@ -20,6 +20,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   [cyberark/conjur-api-python3#89](https://github.com/cyberark/conjur-api-python3/issues/89)
 
 ### Changed
+- CLI command UX has been improved according to UX guidelines
+  [cyberark/conjur-api-python3#132](https://github.com/cyberark/conjur-api-python3/issues/132)
+  See [design guidelines] (https://ljfz3b.axshare.com/#id=x8ktq8&p=conjur_help__init&g=1)
```
Comment (orenbm) [inline CHANGELOG.md:25]: remove space after `]` so the link works

File: CHANGELOG.md (hunk 2 above)
Comment (sigalsax) [reply CHANGELOG.md:14]: the SDK. will update my entries -> SDK

File: CHANGELOG.md [hunk 4]
```diff
@@ -7,19 +7,23 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## Unreleased
 
 ### Added
-- The `host` method rotate-api-key are now available in CLI and API to manage hosts
+- The `host` method rotate-api-key are now available in CLI and SDK to manage hosts
```
Comment (orenbm) [inline CHANGELOG.md:10]: is now

File: CHANGELOG.md [hunk 5]
```diff
@@ -7,19 +7,23 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
   [cyberark/conjur-api-python3#101](https://github.com/cyberark/conjur-api-python3/issues/101)
-- The `user` methods rotate-api-key and change-password are now available in CLI and API to manage users
+- The `user` methods rotate-api-key and change-password are now available in CLI and SDK to manage users
```
Comment (orenbm) [inline CHANGELOG.md:12]: please add `'` for each method

File: CHANGELOG.md (hunk 4 above)
Comment (orenbm) [reply CHANGELOG.md:10]: please add `'` for each method

**Review by tzheleznyak — APPROVED** [review]

**Pull request merged**